backend/src/controllers/CMSTutorialController.ts

```diff
@@ -1,33 +1,33 @@
 import { Request, Response } from "express";
 import { ObjectId, MongoError } from "mongodb";
-import { Subject } from "rxjs";
 import jwt from "jsonwebtoken";
 import Tutorial from "../dtos/Tutorial.dto";
 import PartialTutorial from "../dtos/PartialTutorial.dto";
 import { INTERNAL_ERROR_TEXT, SUCCESSFUL_DOC_CREATION, BAD_REQUEST_TEXT } from "../constants";
-import TutorialDbService from "../services/TutorialDbService";
+import TutorialUpdateService from "../services/TutorialUpdateService";
 
 /**
  * Tutorial route controller
  *
  * @author ale8k
  */
-export default class TutorialController {
+export default class CMSTutorialController {
     /**
      * HTTP method handlers
      */
-    public static get: (req: Request, res: Response) => void = TutorialController.getAllTutorials;
-    public static post: (req: Request, res: Response) => void = TutorialController.createTutorial;
-    public static put: (req: Request, res: Response) => void = TutorialController.updateTutorialById;
+    public static get: (req: Request, res: Response) => void = CMSTutorialController.getAllTutorials;
+    public static post: (req: Request, res: Response) => void = CMSTutorialController.createTutorial;
+    public static put: (req: Request, res: Response) => void = CMSTutorialController.updateTutorialById;
```

Comment:
since, it a tutorial controller and you've followed the single responsibility principle, there's no need to say `getAllTutorials` instead you can simply as `getAll`. It's fine by the way. no need to change.

Reply:
It's just an internal alias, we may have 'getAllCategories' or, 'getAllPartialTutorials' eventually so I think leaving it would be better...?

Reply:
why would you need partial tutorials? Categories will go to category controller.
you can have `getPartials()` in tutotorialcontroller.

Reply:
Then it's not restful, we'd have a get route through with params based on the callback it needs, so 1 method, multiple responses, single endpoint?

Reply:
hmmm... .. in that case its fine.